docs/source/docs/troubleshooting/unix-commands.md

```diff
@@ -0,0 +1,106 @@
+Useful Unix Commands
+====================
+
+SSH
+---
+
+[SSH (Secure Shell)](https://www.mankier.com/1/ssh) is used to securely connect from a local to a remote system (ex. from a laptop to a coprocessor). Unlike other commands on this page, ssh is not Unix specific and can be done on Windows and MacOS from their respective terminals.
+
+:::{note}
+You may see a warning similar to `The authenticity of host 'xxx' can't be established...` or `WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!`, in most cases this can be safely ignored if you have confirmed that you are connecting to the correct host over a secure connection, and the fingerprint will change when your operating system is reinstalled or PhotonVision's coprocessor image is re-flashed. You can read more about it [here](https://superuser.com/questions/421997/what-is-a-ssh-key-fingerprint-and-how-is-it-generated)
+:::
+
+Example:
+
+:::{code-block}
+ssh username@hostname
```

Comment:
Do we want to more explicitly call out photonvision username/hostname etc here? If not, why bother duplicating generic ssh docs?